bots/discord/muse/egg-muse.json

```diff
@@ -0,0 +1,82 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v2",
+        "update_url": null
+    },
+    "exported_at": "2023-08-06T19:26:29+00:00",
+    "name": "Muse",
+    "author": "hello@apollo.moe",
+    "description": "\ud83c\udfa7 a self-hosted midwestern Discord music bot that doesn't suck",
+    "features": null,
+    "docker_images": {
+        "ghcr.io\/software-noob\/pterodactyl-images:nodejs_16": "ghcr.io\/software-noob\/pterodactyl-images:nodejs_16"
+    },
+    "file_denylist": [],
+    "startup": "npm start",
```

Comment:
The bot may start, but can it successfully play a song? There is a [known upstream dependency bug](https://github.com/codetheweb/muse/issues/948#issuecomment-1661423042) that will not allow the latest version to work. Only an older version with a patched dependency will get it to work. If you want, I already have a "working" egg for this, but I was holding off on making a PR for it because it has to run an older version and patch a dependency.